### ISSUE TITLE
Metrics view is broken when locale is not EN-US

### ISSUE THREAD

**Issue by g7ed6e** [dotnet/aspire#1433]
Navigating to the metrics tab of the dashboard results to the exception below when locale is FR-FR
```cmd
warn: Microsoft.AspNetCore.Components.Server.Circuits.RemoteRenderer[100]
      Unhandled exception rendering component: Value cannot be null. (Parameter 'name')
      System.ArgumentNullException: Value cannot be null. (Parameter 'name')
         at System.ArgumentNullException.Throw(String paramName)
         at Microsoft.Extensions.Localization.StringLocalizer`1.get_Item(String name)
         at Aspire.Dashboard.Components.ChartContainer.<>c__DisplayClass49_4.<BuildRenderTree>b__18(RenderTreeBuilder __builder3)
         at Microsoft.AspNetCore.Components.Rendering.RenderTreeBuilder.AddContent(Int32 sequence, RenderFragment fragment)
         at Microsoft.FluentUI.AspNetCore.Components.TemplateColumn`1.CellContent(RenderTreeBuilder builder, TGridItem item) in /_/src/Core/Components/DataGrid/Columns/TemplateColumn.cs:line 25
         at Microsoft.FluentUI.AspNetCore.Components.FluentDataGrid`1.<>c__DisplayClass116_1.<RenderRow>b__2(RenderTreeBuilder __builder) in /_/src/Core/Components/DataGrid/FluentDataGrid.razor:line 100
         at Microsoft.AspNetCore.Components.Rendering.RenderTreeBuilder.AddContent(Int32 sequence, RenderFragment fragment)
         at Microsoft.FluentUI.AspNetCore.Components.FluentDataGrid`1.<>c__DisplayClass116_1.<RenderRow>b__1(RenderTreeBuilder __builder3) in /_/src/Core/Components/DataGrid/FluentDataGrid.razor:line 100
         at Microsoft.AspNetCore.Components.Rendering.RenderTreeBuilder.AddContent(Int32 sequence, RenderFragment fragment)
         at Microsoft.FluentUI.AspNetCore.Components.FluentDataGridCell`1.BuildRenderTree(RenderTreeBuilder __builder) in /_/src/Core/Components/DataGrid/FluentDataGridCell.razor:line 10
         at Microsoft.AspNetCore.Components.Rendering.ComponentState.RenderIntoBatch(RenderBatchBuilder batchBuilder, RenderFragment renderFragment, Exception& renderFragmentException)
fail: Microsoft.AspNetCore.Components.Server.Circuits.CircuitHost[111]
      Unhandled exception in circuit 'XJWudscyAwFHEOjQGcA9DhwqQpPlI3BrTLz-QGdY89U'.
      System.ArgumentNullException: Value cannot be null. (Parameter 'name')
         at System.ArgumentNullException.Throw(String paramName)
         at Microsoft.Extensions.Localization.StringLocalizer`1.get_Item(String name)
         at Aspire.Dashboard.Components.ChartContainer.<>c__DisplayClass49_4.<BuildRenderTree>b__18(RenderTreeBuilder __builder3)
         at Microsoft.AspNetCore.Components.Rendering.RenderTreeBuilder.AddContent(Int32 sequence, RenderFragment fragment)
         at Microsoft.FluentUI.AspNetCore.Components.TemplateColumn`1.CellContent(RenderTreeBuilder builder, TGridItem item) in /_/src/Core/Components/DataGrid/Columns/TemplateColumn.cs:line 25
         at Microsoft.FluentUI.AspNetCore.Components.FluentDataGrid`1.<>c__DisplayClass116_1.<RenderRow>b__2(RenderTreeBuilder __builder) in /_/src/Core/Components/DataGrid/FluentDataGrid.razor:line 100
         at Microsoft.AspNetCore.Components.Rendering.RenderTreeBuilder.AddContent(Int32 sequence, RenderFragment fragment)
         at Microsoft.FluentUI.AspNetCore.Components.FluentDataGrid`1.<>c__DisplayClass116_1.<RenderRow>b__1(RenderTreeBuilder __builder3) in /_/src/Core/Components/DataGrid/FluentDataGrid.razor:line 100
         at Microsoft.AspNetCore.Components.Rendering.RenderTreeBuilder.AddContent(Int32 sequence, RenderFragment fragment)
         at Microsoft.FluentUI.AspNetCore.Components.FluentDataGridCell`1.BuildRenderTree(RenderTreeBuilder __builder) in /_/src/Core/Components/DataGrid/FluentDataGridCell.razor:line 10
         at Microsoft.AspNetCore.Components.Rendering.ComponentState.RenderIntoBatch(RenderBatchBuilder batchBuilder, RenderFragment renderFragment, Exception& renderFragmentException)
```
When locale is set to EN-US Metrics are correctly displayed

**Comment by tlmii** [comment]
@adamint Could this be related to #1440?

**Comment by adamint** [comment]
@tlmii it should be the same issue, but I will confirm

**Comment by adamint** [comment]
Yep, this has been fixed as part of the #1440 pr
